src/issues/util.ts

```diff
@@ -492,10 +492,19 @@ export async function createGithubPermalink(
 		}
 	}
 	const pathSegment = uri.path.substring(repository.rootUri.path.length);
-	const rangeString = range ? `#L${range.start.line + 1}-L${range.end.line + 1}` : '';
+	const rangeString = () => {
```

Comment:
This could theoretically be kept one one line, or shortened by conditionally updating a variable in the main parent block. However, I thought this would be the easiest to read, since the calculation is entirely inside block without side effects.